NCSUVGDC/Lifeworld
# Initial Symptom Manager

* Actors can now be tagged with symptoms
* System self-manages executing actors with symptoms and removing effects when expired

**Testing:**
* open the Test Level and run
* you should see the Flee Test pawn be added to the active symptoms array and lose its symptom after 10 seconds (Note: Flee functionality will be added soon, just prints for now)
* See console log for output
